main/api/routes/user.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+
+from main.api.dependencies.get_db import get_db_session
+from main.models.user import User
+from main.schemas.user import (
+    UserAuthenticationRequestSchema,
+    UserCreationRequestSchema
+)
+from main.schemas.token import AccessToken
+from main.services.auth import create_access_token, verify_password
+from main.services.database.user.crud import create_user, get_user_by_email
+
+router = APIRouter()
+
+
+def authenticate_user(
```

Comment:
Bring back to the controller, no need for a dedicated function as we don't have any intention to reuse